### PR TITLE
[RIP-9]Submit yaml file of deployment type

### PR DESCRIPTION
**Pull request by first-sight12**
Submit yaml test files of deployment type

### CODE REVIEW

File: examples/deployment/deployment-cloudzone.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tcp-echo-cloud
+  template:
+    metadata:
+      labels:
+        app: tcp-echo-cloud
+    spec:
+      nodeName: k8s-node01
```
Comment (Poorunga) [inline examples/deployment/deployment-cloudzone.yaml:18]: We don’t need to specify a `nodeName` anymore.

File: examples/deployment/deployment-cloudzone.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
```
Comment (Poorunga) [inline examples/deployment/deployment-cloudzone.yaml:83]: Same as above.

**Comment by Poorunga** [comment]
Why not make changes to the original file?

File: examples/deployment/deployment-cloudzone.yaml (hunk 2 above)
Comment (first-sight12) [reply examples/deployment/deployment-cloudzone.yaml:83]: OK, I'll revise it right away

Comment (first-sight12) [reply examples/deployment/deployment-cloudzone.yaml:83]: If the node name is not specified, the pod will be dispatched to the edge node。
image

Comment (Poorunga) [reply examples/deployment/deployment-cloudzone.yaml:83]: Then we should try to use node affinity.

Comment (first-sight12) [reply examples/deployment/deployment-cloudzone.yaml:83]: OK, I'll modify it locally and test it

Comment (first-sight12) [reply examples/deployment/deployment-cloudzone.yaml:83]: So, should I label the cloud nodes myself,

The following is the label information of my node node

image

**Comment by first-sight12** [comment]
How should I operate here
image

File: examples/deployment/deployment-cloudzone.yaml (hunk 2 above)
Comment (Poorunga) [reply examples/deployment/deployment-cloudzone.yaml]: You can refer to https://github.com/kubeedge/edgemesh/blob/main/build/agent/kubernetes/edgemesh-agent/07-daemonset-cloud.yaml

```
...
   affinity:
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
              - matchExpressions:
                  - key: node-role.kubernetes.io/edge
                    operator: DoesNotExist
                  - key: node-role.kubernetes.io/agent
                    operator: DoesNotExist
...
```

Comment (first-sight12) [reply examples/deployment/deployment-cloudzone.yaml]: OK, I'll modify it and test it

**Comment by first-sight12** [comment]
I have finished the modification. Do you have anything else to modify
@Poorunga

**Pull request closed without merging**